### PR TITLE
Bump redhat-hll to 2.18

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

**Review by hanefi — CHANGES_REQUESTED** [review]
I think we should remove PG10 from our builds and remove references to it in pg_exclude.yml

**Comment by gurkanindibay** [comment]
removed pg 10

File: pg_exclude.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 exclude:
   nightly:
     ol/7: [15]
+    el/8: [10]
```
Comment (hanefi) [inline pg_exclude.yml]: Why do we still need exclusion rules for PG10?

Comment (gurkanindibay) [reply pg_exclude.yml]: Kept it in case of previous minors but not needed as you suggested

File: postgres-matrix.yml [hunk 2]
```diff
@@ -5,4 +5,6 @@ version_matrix:
   - 2.16:
       postgres_versions: [ 10, 11, 12, 13, 14 ]
   - 2.17:
-      postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
+      postgres_versions: [ 10, 11, 12, 13, 14, 15, 16 ]
```
Comment (hanefi) [inline postgres-matrix.yml]: It does not make sense to add PG16 support to 2.17. 2.18 should be the first version to support this version of PostgreSQL.

```suggestion
      postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
```

**Review by hanefi — APPROVED** [review]

**Pull request merged**